Description: Bump toml version from 0.7 to 0.8
Author: Facundo Gaich <facugaich@gmail.com>
Last-Update: 2024-02-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: toml2json/Cargo.toml
===================================================================
--- toml2json.orig/Cargo.toml
+++ toml2json/Cargo.toml
@@ -47,4 +47,4 @@ features = ["derive"]
 version = "1.0"
 
 [dependencies.toml]
-version = "0.7"
+version = ">= 0.7, < 0.9"
